### ISSUE TITLE
Run on conventional Perturb-seq

### ISSUE THREAD

**Issue by kiddo18** [douglasyao/FR-Perturb#1]
Hi! Just wondering if FR-Perturb can be run on 1-guide 1-cell conventional Perturb-seq dataset?

Also do you have any suggestions for plotting UMAP for cell-guide pair that works/gets good perturbation efficiency?

**Comment by karlie002** [comment]
Hi, have you gotten the answer?

**Comment by douglasyao** [comment]
Yes, FR-Perturb can be run on 1-guide 1-cell conventional Perturb-seq - in
fact, we make the claim that it outperforms other methods for these
datasets.

The method has been recently updated so I recommend updating to the latest
version.
